tee/src/lib.rs

```diff
@@ -374,6 +376,19 @@ pub mod pallet {
 			Ok(().into())
 		}
 
+
+		/// Remove an enclave update request from storage
+		#[pallet::weight(T::WeightInfo::remove_update_request())]
+		pub fn remove_update_request(
+			origin: OriginFor<T>,
+			operator_address: T::AccountId,
+		) -> DispatchResultWithPostInfo {
+			ensure_root(origin)?;
+			EnclaveUpdates::<T>::remove(operator_address.clone());
```

Comment:
I think we should do a check whether update request exist or not before removing it, no? if it doesnt exist we should return an eroor.

Reply:
no error happens @Immanuel-john

Reply:
No error is being thrown if the key is not available @Immanuel-john